src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,41 @@
 import React from 'react';
+
+import { TypeMovie } from '../../types';
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <div className="card" key={movie.imdbId}>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={movie.imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="./images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8">{movie.title}</p>
+        </div>
+      </div>
+
+      <div className="content">
+        {movie.description}
+        <br />
+        <a href={movie.imdbUrl}>IMDB</a>
+      </div>
+    </div>
+  </div>
 );
+
+MovieCard.propTypes = {
```

Comment:
Since `TypeMovie` is a `shape` (`object`), I'm not sure if this syntax is correct. Maybe you could write just like this: `MovieCard.propTypes = TypeMovie.isRequired`. But I'm not sure.

Reply:
no, it's okay